semcore/side-panel/src/SidePanel.jsx

```diff
@@ -213,16 +213,14 @@ function Footer(props) {
 function Close({ styles, children: hasChildren, Children, getI18nText }) {
   const SClose = Root;
   return sstyled(styles)(
-    <SClose
-      render={Button}
-      aria-label={getI18nText('close')}
-      use='tertiary'
-      theme='muted'
-      type='button'
-      size='l'
-      addonLeft={hasChildren ? undefined : CloseIcon}
-    >
-      {hasChildren ? <Children /> : null}
+    <SClose render={Button} aria-label={getI18nText('close')} use='tertiary' theme='muted' size='l'>
```

Comment:
What the point of this change?

Reply:
we don't need `type=button` because this is default.

we need to add margins to `Button.Addon`